src/phaser-game/scenes/FortNerf.ts

```diff
@@ -173,6 +193,17 @@ class FortNerf extends Phaser.Scene {
         player.anims.play('rightstill', true);
       }
     }
+    if (cursors.space.isDown) {
```

Comment:
Maybe add a setTimeout() to the shootBullet so it doesn't look like graffiti? Not entirely sure how it works. Cool effect though.